src/sentry/static/sentry/app/components/activity/note/input.jsx

```diff
@@ -375,23 +379,166 @@ const NoteInput = createReactClass({
               />
             </MentionsInput>
           )}
-          <div className="activity-actions">
-            {errorMessage && <small className="error">{errorMessage}</small>}
-            <button className="btn btn-default" type="submit" disabled={loading}>
-              {btnText}
-            </button>
+        </NoteInputBody>
+
+        <Footer>
+          <div>{errorMessage && <ErrorMessage>{errorMessage}</ErrorMessage>}</div>
+          <div>
             {updating && (
-              <button className="btn btn-danger" onClick={this.onCancel}>
+              <FooterButton priority="danger" type="button" onClick={this.onCancel}>
                 {t('Cancel')}
-              </button>
+              </FooterButton>
             )}
+            <FooterButton error={errorMessage} type="submit" disabled={loading}>
+              {btnText}
+            </FooterButton>
           </div>
-        </div>
-      </form>
+        </Footer>
+      </NoteInputForm>
     );
   },
 });
 
 export {NoteInput};
 
 export default withApi(NoteInput);
+
+const notePreviewCss = css`
+  display: block;
+  width: 100%;
+  min-height: 140px;
+  max-height: 1000px;
+  max-width: 100%;
+  margin: 0;
+  border: 0;
+  padding: 15px 20px 0;
```

Comment:
Should we be using space() to normalize padding?

Reply:
I wanted to keep the old values for now (at least for padding) - I'll see how radius looks though

Reply:
Ah I just ended up changing it because